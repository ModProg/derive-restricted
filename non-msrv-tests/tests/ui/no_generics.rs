#[derive(derive_where::DeriveWhere)]
#[derive_where(Clone)]
struct Test1(u8);

#[derive(derive_where::DeriveWhere)]
#[derive_where(Clone)]
enum Test2 {
	A(u8),
}

fn main() {}
