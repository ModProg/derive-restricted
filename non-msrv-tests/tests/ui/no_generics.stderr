error: derive-where doesn't support items without generics, as this can already be handled by standard `#[derive()]`
 --> tests/ui/no_generics.rs:2:1
  |
2 | struct Test1(u8);
  | ^^^^^^^^^^^^^^^^^

error: derive-where doesn't support items without generics, as this can already be handled by standard `#[derive()]`
 --> tests/ui/no_generics.rs:5:1
  |
5 | enum Test2 { A }
  | ^^^^^^^^^^^^^^^^

error: derive-where doesn't support items without generics, as this can already be handled by standard `#[derive()]`
  --> tests/ui/no_generics.rs:10:1
   |
10 | struct Test3();
   | ^^^^^^^^^^^^^^^

error: derive-where doesn't support items without generics, as this can already be handled by standard `#[derive()]`
  --> tests/ui/no_generics.rs:13:1
   |
13 | enum Test4 { }
   | ^^^^^^^^^^^^^^
